### DISCUSSION TITLE
Not all extensions are available in the demo, how to enable that behavior?

### DISCUSSION THREAD

**Discussion by invm** [CodinGame/monaco-vscode-api#454] [Q&A] [answered]
What am I missing?

**Comment by CGNonofr** [comment]
Can you provide a little more details? what are you talking about? the default VSCode extension? the marketplace?

**Reply by invm** [reply]
When running the demo, and going to extensions marketplace, searching for 'settings sync', I'm seeing this message:
![image](https://github.com/CodinGame/monaco-vscode-api/assets/52990657/a61eb59e-54ee-45ea-8336-274e29057a53)

**Reply by CGNonofr** [reply]
The "Learn Why" link explain it well

The extension needs to be web compatible, which is not the case for this one.

The only option you have is to run the extension host server and connect to it to be able to run any extension

**Reply by invm** [reply]
Can you point to the extension host server documentation? Other than
```js
import 'vscode/localExtensionHost'
````

What do I need to do?

**Reply by CGNonofr** [reply, marked as answer]
That is not the extension host server, that's the local extension host.

The doc you're looking for is https://github.com/CodinGame/monaco-vscode-api/blob/main/docs/vscode_server.md

**Reply by invm** [reply]
Thanks, thats it.
